azure-pipelines.yml

```diff
@@ -72,7 +72,7 @@ jobs:
       displayName: 'Use Python 3.6'
       inputs:
         versionSpec: 3.6
-    - bash: pip install wheel==0.30.0 pylint==1.9.5 flake8==3.5.0 requests
+    - bash: pip install wheel pylint==1.9.5 flake8==3.5.0 requests pkginfo
```

Comment:
Not considering breaking change versions for wheel and pkginfo?

Reply:
As for wheel, I am not wrong about the breaking change unless the PEP related the Python Package. The latest and unresolved PEP about this is [PEP 491](https://www.python.org/dev/peps/pep-0491/), but it seems won'
t introduce any breaking change, it's an enhancement.

As for pkginfo, I am a bit concerned about it, because it's onboard on Ubuntu Launch Code with little issues and invisible about the development activity. 

So, if we relax the majon version of wheel and pkginfo can't keep up with the latest PEP version, that's a risk.

But considering the PEP process, it won't be a problem for a long while.